simplic-interval/Simplic.Interval.Service/IntervalService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Dapper;
+using Simplic.Framework.DAL.DatabaseMetadata;
+using Simplic.Sql;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Simplic.Interval.Service
+{
+    public class IntervalService : IIntervalService
+    {
+        #region Fields
+
+        private readonly ISqlService sqlService;
+
+        #endregion Fields
+
+        #region Constructor
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="sqlService"></param>
+        public IntervalService()
+        {
+            this.sqlService = CommonServiceLocator.ServiceLocator.Current.GetInstance<Simplic.Sql.ISqlService>();
+        }
+
+        #endregion Constructor
+
+        #region Public methods
+
+        /// <summary>
+        /// Gets all available configured intervals
+        /// </summary>
+        /// <returns>Intervals</returns>
+        public IEnumerable<Interval> GetAll()
+        {
+            return sqlService.OpenConnection((connection) =>
+            {
+                return connection.Query<Interval>("Select * from IT_Interval");
+            });
+        }
+
+        /// <summary>
+        /// Gets a single interval
+        /// </summary>
+        /// <param name="intervalId"></param>
+        /// <returns>Interval</returns>
+        public Interval Get(Guid intervalId)
+        {
+            return sqlService.OpenConnection((connection) =>
+            {
+                return connection.Query<Interval>("Select * from IT_Interval where Guid = :id",new {id= intervalId }).SingleOrDefault();
```

Comment:
check spaces between characters. also dont use `SingleOrDefault()`, instead use `FirstOrDefault()` . `SingleOrDefault()` throws an exception if there are more than one record returned.

Reply:
if there is more then one record , i want an exception because this is critical :)